libpromises/iteration.c

```diff
@@ -334,7 +334,6 @@ static void MangleVarRefString(char *ref_str, size_t len)
         *ns      = CF_MANGLED_NS;
         ref_str2 =  ns + 1;
         upto    -= (ns + 1 - ref_str);
-        assert(upto >= 0);
```

Comment:
This is a compiler warning, not a user facing change. Please remove the changelog entry. (`Changelog: None`).